test/wallet-test.js

```diff
@@ -2463,8 +2459,8 @@ describe('Wallet', function() {
       let bal = await wallet.getBalance();
       assert.strictEqual(bal.tx, 2);
       assert.strictEqual(bal.coin, 2);
-      assert.strictEqual(bal.confirmed, 10e6 - (1 * fee));
-      assert.strictEqual(bal.unconfirmed, 10e6 - (1 * fee));
+      assert.strictEqual(bal.confirmed, 10e6 - (cTXCount * fee));
+      assert.strictEqual(bal.unconfirmed, 10e6 - (uTXCount * fee));
```

Comment:
sorry to be annoying about this, but I was thinking the counter variable could replace the integer in all the test blocks like [in my example diff](https://github.com/handshake-org/hsd/pull/656#discussion_r761390391) what youve got here is fine to get around the implicit coercion error but the full diff might make the test more readable.

Reply:
> sorry to be annoying about this, but I was thinking the counter variable could replace the integer in all the test blocks like [in my example diff](https://github.com/handshake-org/hsd/pull/656#discussion_r761390391) what youve got here is fine to get around the implicit coercion error but the full diff might make the test more readable.

I believe I just saw: maybe we can remove implicit coercion - and just went removing it w/o looking at diff, totally missed that one.. Sorry !